README.md

```diff
@@ -1 +1,25 @@
-# hw01
\ No newline at end of file
+#*HW01 - Adam Berger *
+
+##Biography
+
+My name is Adam Berger and I am a fourth-year student in the College majoring in *Public Policy* and minoring in *Near Eastern Languages and Civilizations*. I am originally  from Bainbridge, Ohio, a small town about 45-minutes outside of Cleveland. On campus, I work as a tour guide in the Office of College Admissions and proudly serve as the RA of Alper House in Max Palevsky East.
+
+![Here's what I look like, featuring my Alper gear](IMG_2151.jpg)
```

Comment:
I am not entirely sure why the picture is not showing. Your syntax should be correct. Maybe try simplifying image title/image name?